ecommerce/serializers/subcategories.py

```
@@ -0,0 +1,112 @@
+        read_only_fields = [
+            "id",
+            "slug",
+            "category",
+            "created_at",
+            "updated_at",
+        ]
+
+    def get_category(self, obj):
+        """Get all active subcategories of this category."""
+        # Import here to avoid circular import
+        from .categories import CategorySimpleSerializer
```

Ruff (0.12.2)

`import` should be at the top-level of a file

(PLC0415)